### PR TITLE
Added unit test for Select

### LINKED ISSUE #933: Add tests for Select

### PR DESCRIPTION
**Pull request by AJIL-PAUL**
Fixes #933 

- Added unit test for Select component

### CODE REVIEW

**Comment by AJIL-PAUL** [comment]
@ajmaln _a Please review

File: tests/Select.test.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  it("should render without error", () => {
+    const { getByText } = render(<Select label="Select" options={options} />);
+    expect(getByText("Select")).toBeInTheDocument();
+  });
+
+  it("should show option list on clicking", () => {
+    const { container, getByText } = render(
+      <Select label="Select" options={options} />
+    );
+    userEvent.click(container.querySelector("input"));
```
Comment (ajmaln) [inline tests/Select.test.js:27]: Prefer using `getByRole` or `getByLabelText` instead of `container.querySelector`

**Review by ajmaln — COMMENTED** [review]
@AJIL-PAUL _a Please check the suggestions. Also could you cover the creatable select feature as well.

File: tests/Select.test.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    expect(getByText("Option 1")).toBeInTheDocument();
+    expect(getByText("Option 2")).toBeInTheDocument();
+  });
+
+  it("should call onChange on select option", () => {
+    const onChange = jest.fn();
+    const { container, getByText } = render(
+      <Select label="Select" options={options} onChange={onChange} />
+    );
+    userEvent.click(container.querySelector("input"));
```
Comment (ajmaln) [inline tests/Select.test.js:37]: Prefer using `getByRole` or `getByLabelText` instead of `container.querySelector`

**Comment by AJIL-PAUL** [comment]
Updated unit test for Select

@ajmaln _a Please review

File: tests/Select.test.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    expect(screen.getByText("Option 2")).toBeInTheDocument();
+  });
+
+  // it("should be disabled if isDisabled is true", () => {
+  //   const { getByRole } = render(
+  //     <Select label="Select" options={options} isDisabled />
+  //   );
+  //   const select = getByRole("combobox");
+  //   expect(select).toBeDisabled();
+  // });
```
Comment (ajmaln) [inline tests/Select.test.js:57]: Remove this if not required

File: tests/Select.test.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  {
+    label: "Option 2",
+    value: "option-2",
+  },
+];
+
+describe("Select", () => {
+  it("should render without error", () => {
+    render(<Select label="Select" options={options} />);
+    const select = screen.getAllByRole("combobox");
```
Comment (ajmaln) [inline tests/Select.test.js:20]: Use `getByRole` instead

**Review by ajmaln — COMMENTED** [review]
@AJIL-PAUL _a Please check the suggestions

**Comment by AJIL-PAUL** [comment]
- Updated unit test based on Select

@ajmaln _a Please review

**Review by ajmaln — APPROVED** [review]

**Pull request merged**